### DISCUSSION TITLE
Setting indexed properties exposed by COM result in error for strict mode.

### DISCUSSION THREAD

**Discussion by pontusn** [MicrosoftEdge/WebView2Feedback#3864] [Q&A]
TypeError: 'set' on proxy: trap returned falsish for property

**Comment by champnic** [comment]
Can you add some details about what you are doing here? Could you share some code and where the error is happening? Thanks!

**Comment by pontusn** [comment]
I will try to provide more details. The problem was seen when setting indexed properties using sync proxy objects from injected JavaScript in response to onclick. The same properties was successfully set from the console.

Our environment is quite complex. I will try to find time to reproduce separately.

**Comment by champnic** [comment]
Even if not a repro, if you could share the code around where the error happens that would be helpful.

**Comment by pontusn** [comment]
I think this issue is highly related to #567

**Comment by pontusn** [comment]
Failed to repro in new project until I happened use propputref...
